test/realm-fuzzer/.gitignore

```diff
@@ -0,0 +1,5 @@
+findings/*
```

Comment:
Why would we want this file? We don't want to run/build the fuzzer from this directory

Reply:
Yep. I agree, It must be something I forgot.